README.md

```diff
@@ -45,6 +45,21 @@ When you design beautiful block patterns for your theme, Pattern Manager by WP E
  * Create/update the pattern PHP file on your disk, in your theme.
  * Find and replace all local image urls with the correct PHP tag.
  * Find and copy all image files used into your theme.
+ 
+## Additional Features
+
+### The Pattern Block: Put patterns inside patterns.
+Once you've built a lot of patterns, it makes sense to quickly combine them in various ways to make full-page patterns. An example of this might be a "Home Page" or "Landing Page" pattern that includes a call-to-action area, features area, testimonials, links, and more. Instead of re-creating each of those patterns inside your "Home Page" pattern, you can simply re-use them with the "Pattern Block".
```

Comment:
This nails it:

>Instead of re-creating each of those patterns inside your "Home Page" pattern, you can simply re-use them with the "Pattern Block".